main.py

```diff
@@ -352,13 +475,18 @@ def sigint_handler(signum, frame):
         TargetHostTest('api.build02.gcp.ci.openshift.org', TestType.DNS_TCP_LOOKUP, None),
         TargetHostTest('api.build04.34d2.p2.openshiftapps.com', TestType.DNS_TCP_LOOKUP, None),
         TargetHostTest('static.redhat.com', TestType.DNS_TCP_LOOKUP, None),
+
+        TargetHostTest('', TestType.LIVENESS_PROBE_ICMP, None),
+        TargetHostTest('', TestType.LIVENESS_PROBE_UDP, None),
+        # TargetHostTest('', TestType.LIVENESS_PROBE_TCP, None),
     ]
     signal.signal(signal.SIGINT, sigint_handler)
 
     in_q = queue.Queue()
 
     for test in tests_to_run:
-        threading.Thread(target=monitor_host, args=(test,)).start()
+        threading.Thread(target=monitor_host, args=(test, delay)).start()
+        # monitor_host(test, delay)
```

Comment:
is this supposed to be here, or artifact from testing?

Reply:
Ah, that is for easier testing locally :-)